src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -47,6 +48,11 @@ public function all(): array
         return $this->parameters;
     }
 
+    public function allDeprecated(): array
```

Comment:
should this be marked as `@internal` if you don't consider it part of the public API ? For now, it is public.

Reply:
I think it's fine having it public.

Reply:
Maybe we should either mark the `ParameterBag` new methods as internal indeed or document them in the changelog if they are public?

Reply:
Fine as-is to me, no need to talk about it

Reply:
Since I now use protected (https://github.com/symfony/symfony/pull/47719#issuecomment-1356260382 and https://github.com/symfony/symfony/pull/47719/commits/63d39b0ac7e93ffb5d74e26e53652035cc49c634), maybe we should at least mark the methods as final?
Btw the `FrozenParameterBag` class should also be marked `@final` as of 6.3, wdyt?

Reply:
I'm fine as is personally. Making it final won't provide us anything.